PROTOCOL.md

```diff
@@ -424,6 +429,66 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests can be sent with `|RQID`. `RQID` is a unique number used to
```

Comment:
"Most" is distressingly nonspecific. Just say "all except `/undo`". Also be clear that `|RQID` is added to the end.